beacon-chain/sync/initial-sync-old/round_robin.go

```diff
@@ -209,10 +219,12 @@ func (s *Service) roundRobinSync(genesis time.Time) error {
 
 		for _, blk := range blocks {
 			s.logSyncStatus(genesis, blk.Block, peers, counter)
-			if !s.db.HasBlock(ctx, bytesutil.ToBytes32(blk.Block.ParentRoot)) {
-				log.Debugf("Beacon node doesn't have a block in db with root %#x", blk.Block.ParentRoot)
+			parentRoot := bytesutil.ToBytes32(blk.Block.ParentRoot)
+			if !s.db.HasBlock(ctx, parentRoot) && !s.chain.HasInitSyncBlock(parentRoot) {
+				log.Debugf("Beacon node doesn't have a block in db or cache with root %#x", parentRoot)
```

Comment:
Should this be changed to `WithField()` for better log tracking?

Reply:
Also, not feature flag needed here?

Reply:
Updated to WithField(), feature flag not needed in this csae